corehq/apps/data_interfaces/interfaces.py

```diff
@@ -141,6 +145,43 @@ def bulk_response(self):
         )
 
 
+@location_safe
+class CaseCopyInterface(CaseReassignmentInterface):
+    name = gettext_noop("Copy Cases")
+    slug = "copy_cases"
+    report_template_path = 'data_interfaces/interfaces/case_management.html'
+    action = "copy"
+    action_text = gettext_lazy("Copy")
+
+    @property
+    @memoized
+    def es_results(self):
+        query = self._build_query()
+        owner_id = self.request.GET.get('individual')
```

Comment:
Where is this GET param being used? 

nit: any reason to not call the GET param `owner_id`?

Reply:
@sravfeyn 
> Where is this GET param being used?

I'm not sure I know exactly what you're asking?